tests/integration/base_file_test.go

```diff
@@ -60,7 +61,7 @@ func runFileE2ETest(helper *clients.TestClient, k8sh *utils.K8sHelper, s suite.S
 	logger.Infof("Step 4: Read from file system")
 	read, rdErr := helper.FSClient.Read(filePodName, fileMountPath, "fsFile1", namespace)
 	require.Nil(s.T(), rdErr)
-	require.Contains(s.T(), read, "Smoke Test Data for file system storage", "make sure content of the files is unchanged")
+	assert.Contains(s.T(), read, "Smoke Test Data for file system storage", "make sure content of the files is unchanged")
```

Comment:
why the change from require to assert?

Reply:
this is really a non-fatal error and should not block the test from continuing. It masked other failures that I could have fixed at the same time.